### PR TITLE
Additional property descriptions and README files

### LINKED ISSUE #33: Catalogue all known properties
[This script](https://github.com/whosonfirst/whosonfirst-tests/blob/master/test_runs/produce_all_props.py) was created to crawl the `whosonfirst-data` repository to produce [a list](https://github.com/whosonfirst/whosonfirst-tests/blob/master/output/all_props.csv) of all existing properties in WOF records.

Organize this list, pull out anything we should not include (see: the various `zs:` props), and backfill each markdown file in this repository to include these property names.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #33.

Using the text file in #33, I catalogued the known properties in existing WOF records. This PR does not include descriptions for source-specific properties like `can-calcai` (used for Calgary neighbourhoods) and `lacity` (used for LA neighbourhoods).

All other source properties are now catalogued in unique json files.

### CODE REVIEW

File: properties/meso/adm0.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1158846279,
+    "name": "adm0",
+    "prefix": "meso",
+    "description": "",
```
Comment (nvkelso) [inline properties/meso/adm0.json]: Country name

File: properties/meso/adm1.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1158846295,
+    "name": "adm1",
+    "prefix": "meso",
+    "description": "",
```
Comment (nvkelso) [inline properties/meso/adm1.json]: Region name

File: properties/meso/adm1_alt.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1158846281,
+    "name": "adm1_alt",
+    "prefix": "meso",
+    "description": "",
```
Comment (nvkelso) [inline properties/meso/adm1_alt.json]: Alternative region name

File: properties/meso/adm1_alt2.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1158846283,
+    "name": "adm1_alt2",
+    "prefix": "meso",
+    "description": "",
```
Comment (nvkelso) [inline properties/meso/adm1_alt2.json]: Alternative alternate region name

File: properties/meso/adm1_en.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1158846285,
+    "name": "adm1_en",
+    "prefix": "meso",
+    "description": "",
```
Comment (nvkelso) [inline properties/meso/adm1_en.json]: Region name in English

File: properties/meso/adm1_loc.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1158846287,
+    "name": "adm1_loc",
+    "prefix": "meso",
+    "description": "",
```
Comment (nvkelso) [inline properties/meso/adm1_loc.json]: Region name in local language

File: properties/meso/adm1_loc2.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
+    "id": 1158846289,
+    "name": "adm1_loc2",
+    "prefix": "meso",
+    "description": "",
```
Comment (nvkelso) [inline properties/meso/adm1_loc2.json]: Alternative region name in local language

**Comment by stepps00** [comment]
(Tagging this as WIP)

**Pull request merged**